### ISSUE TITLE
Customizable certificates

### ISSUE THREAD

**Issue by billtomczak** [OSTraining/OSCampus#265]
Change hardcoded OSTraining certificates to a system for custom, user-defined certificates.

@htmgarcia @stevejburge -- we'll want to have some kind of generic certificate image to use for this.

**Comment by htmgarcia** [comment]
I can work on a generic image.

**Comment by billtomczak** [comment]
Thanks @htmgarcia!

**Comment by htmgarcia** [comment]
I attach an HD version:
![update the certificate image-1 0 5](https://cloud.githubusercontent.com/assets/4999794/15677366/a07cba8c-270f-11e6-8526-ef86e47358e5.jpg)

**Comment by stevejburge** [comment]
@htmgarcia @billtomczak Won't this need to have the logo removed? This will be for the public version of OSCampus

**Comment by htmgarcia** [comment]
@stevejburge you're right! So, we will need two images: a default one (generic with no logo) and a custom one (image above)

**Comment by htmgarcia** [comment]
For OSTraining:
![ost](https://cloud.githubusercontent.com/assets/4999794/15678124/bb7e3038-2712-11e6-839d-f7bf526c92e1.jpg)

*********

Generic:
![generic](https://cloud.githubusercontent.com/assets/4999794/15678133/c64eede0-2712-11e6-8048-ee94a92b61a5.jpg)
